src/dev/build/lib/fs.js

```diff
@@ -174,12 +174,14 @@ export async function untar(source, destination, extractOptions = {}) {
   assertAbsolute(source);
   assertAbsolute(destination);
 
+  await mkdirpAsync(destination);
+
   await createPromiseFromStreams([
     fs.createReadStream(source),
     createGunzip(),
-    new Extract({
+    tar.extract({
       ...extractOptions,
-      path: destination
+      cwd: destination
```

Comment:
`path` is no longer an option, and `cwd` according to the docs (and testing) requires that the destination already exists before performing the extraction. Hence the addition of `await mkdirpAsync(destination);`